### PR TITLE
refactor(gwe-est): remove packagedata block and put values in options block

### PR DESCRIPTION
**Pull request by emorway-usgs**
- [x] Added new test or modified an existing test
- [x] Ran `black` on new or modified autotests
- [x] Formatted new and modified Fortran source files with `fprettify`
- [x] Updated [definition files](/MODFLOW-USGS/modflow6/tree/develop/doc/mf6io/mf6ivar)
- [x] Updated [develop.tex](/MODFLOW-USGS/modflow6/doc/ReleaseNotes/develop.tex) with a plain-language description of the bug fix, change, feature; required for changes that may affect users
- [x] Updated [input and output guide](/MODFLOW-USGS/modflow6/doc/mf6io)

For additional information see [instructions for contributing](/MODFLOW-USGS/modflow6/.github/CONTRIBUTING.md) and [instructions for developing](/MODFLOW-USGS/modflow6/.github/DEVELOPER.md).

### CODE REVIEW

File: doc/mf6io/mf6ivar/examples/gwe-est-example.dat [hunk 1]
```diff
@@ -1,5 +1,7 @@
 BEGIN OPTIONS
-  LATENT_HEAT_VAPORIZATION
+  CPW   4180.0
```
Comment (langevin-usgs) [inline doc/mf6io/mf6ivar/examples/gwe-est-example.dat:2]: Should we spell this out as HEAT_CAPACITY_WATER?

File: doc/mf6io/mf6ivar/examples/gwe-est-example.dat [hunk 2]
```diff
@@ -1,5 +1,7 @@
 BEGIN OPTIONS
-  LATENT_HEAT_VAPORIZATION
+  CPW   4180.0  
+  RHOW   999.0
```
Comment (langevin-usgs) [inline doc/mf6io/mf6ivar/examples/gwe-est-example.dat:3]: Spell this one out too?  DENSITY_WATER

**Review by langevin-usgs — COMMENTED** [review]
Thanks for doing this.  I think this is a cleaner implementation.  My only comment would be to consider renaming the keywords to HEAT_CAPACITY_WATER and DENSITY_WATER.  You can leave the internal fortran variables the same (by setting mf6internal in the definition file entries).

File: doc/mf6io/mf6ivar/examples/gwe-est-example.dat (hunk 1 above)
Comment (emorway-usgs) [reply doc/mf6io/mf6ivar/examples/gwe-est-example.dat:2]: Change implemented in 4358e1b

File: doc/mf6io/mf6ivar/examples/gwe-est-example.dat (hunk 2 above)
Comment (emorway-usgs) [reply doc/mf6io/mf6ivar/examples/gwe-est-example.dat:3]: Change implemented in 4358e1b

**Comment by emorway-usgs** [comment]
> ... consider renaming the keywords to HEAT_CAPACITY_WATER and DENSITY_WATER. You can leave the internal fortran variables the same (by setting mf6internal in the definition file entries).

I'm fine with this change, I considered longer names to begin with but opted for the shorter names initially.  Thanks for the suggestion.

File: doc/mf6io/mf6ivar/examples/gwe-est-example.dat (hunk 1 above)
Comment (langevin-usgs) [reply doc/mf6io/mf6ivar/examples/gwe-est-example.dat:2]: Maybe I'm wrong but I don't think gwe-est-example.dat was updated with the new keywords.

**Review by langevin-usgs — COMMENTED** [review]
Hey @emorway-usgs, just a few more comments to think about.

File: doc/mf6io/mf6ivar/dfn/gwe-est.dfn [hunk 3]
```diff
@@ -16,13 +16,33 @@ optional true
...
+default_value 1000.0
+mf6internal rhow
+
+block options
+name heat_capacity_water
+type double precision
+reader urword
+optional true
+longname heat capacity of water
+description heat capacity of water used by calculations related to heat storage and conduction.  This value is set to 4,184 kJ/(kg*C) if no overriding value is specified.  A user-specified value should be provided for models that use units other than kilograms, kilojoules, and degrees Celsius or it is necessary to use a value other than the default.
```
[the remark this answers is not in the document]
Comment (emorway-usgs) [reply doc/mf6io/mf6ivar/dfn/gwe-est.dfn]: Fixed with [209f7d4](https://github.com/MODFLOW-USGS/modflow6/pull/1882/commits/209f7d4429a23600bec6aa8f8981cd21b0941a0b)

File: doc/ReleaseNotes/develop.tex [hunk 4]
```diff
@@ -24,7 +24,7 @@
...
 		\item For ASCII input files erroneously containing a mix of line endings, MODFLOW would sometimes proceed with unexpected results. The program was corrected to stop with an error message if an input line contained both carriage returns and line feeds.
-	%	\item xxx
+		\item A refactor of the energy storage and transfer (EST) package associated with the GWE model type results in different input requirements that breaks backward compatibility with what was required in version 6.5.0.  The PACKAGEDATA block was removed from the EST package input.  In its place, the heat capabity of water, the specified density of water, and the latent heat of vaporization are instead given default values that can be overridden by specifying alternative values in the OPTIONS block.
```
[the remark this answers is not in the document]
Comment (emorway-usgs) [reply doc/ReleaseNotes/develop.tex]: Thanks for catching that, committed your suggestion directly on github.

File: doc/mf6io/mf6ivar/dfn/gwe-est.dfn [hunk 5]
```diff
@@ -16,13 +16,33 @@ optional true
...
 reader urword
 optional true
-longname activate cooling associated with evaporation
-description is a text keyword to indicate that cooling associated with evaporation will occur.  Use of this keyword requires that LATHEATVAP are specified in the GRIDDATA block.  While the EST package does not simulate evaporation, multiple other packages in a GWE simulation may.  For example, evaporation may occur from the surface of streams or lakes.  Owing to the energy consumed by the change in phase, the latent heat of vaporization is required.
+longname latent heat of vaporization
+description latent heat of vaporization is the amount of energy that is required to convert a given quantity of liquid into a gas and is associated with evaporative cooling.  While the EST package does not simulate evaporation, multiple other packages in a GWE simulation may.  To avoid having to specify the latent heat of vaporization in multiple packages, it is specified in a single location and accessed wherever it is needed.  For example, evaporation may occur from the surface of streams or lakes and the energy consumed by the change in phase would be needed in both the SFE and LKE packages.
```
[the remark this answers is not in the document]
Comment (emorway-usgs) [reply doc/mf6io/mf6ivar/dfn/gwe-est.dfn]: Great catch.  Cleaned-up in f5e528b

**Review by langevin-usgs — APPROVED** [review]

File: doc/mf6io/mf6ivar/examples/gwe-est-example.dat (hunk 1 above)
Comment (emorway-usgs) [reply doc/mf6io/mf6ivar/examples/gwe-est-example.dat]: Updated with 33853d6

**Pull request merged**